persistent-template.yml

```diff
@@ -1191,3 +1191,125 @@ objects:
           volumeMounts:
           - name: alarmprovider-config
             mountPath: "/alarmprovider_config"
+# START API Gateway       
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: apigateway
+  data:
+    apimgmt-init: |
```

Comment:
This script isn't referenced by anything. Can we get rid of it? If not, let's remove the redundancy and factor the common stuff into a shared script